Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -199,8 +199,13 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
         .take(first: 1)
     )
 
-    self.projectImageUrl = project
-      .map { URL(string: $0.photo.full) }
+    self.projectImageUrl = Signal.merge(
+      configData.mapConst(nil),
+      Signal.combineLatest(
+        project.map { URL(string: $0.photo.full) },
+        configData.ignoreValues()
+      ).map(first)
+    )
```

Comment:
So I initially had it that way (with viewDidLoad), and just tested it again, bizarrely:

<img width="841" alt="screen shot 2017-01-31 at 9 26 00 pm" src="https://cloud.githubusercontent.com/assets/3735375/22480996/183ceb3e-e7fc-11e6-9c2c-f939651e5da4.png">

```
    self.projectImageUrl = Signal.merge(
      self.viewDidLoadProperty.signal.mapConst(nil),
      project.map { URL(string: $0.photo.full) }
    )
```

I'll debug a little more but will probably just remove it from the story board 😄